cloudify_cli/commands/ssh.py

```diff
@@ -28,6 +30,17 @@
 from cloudify_cli.exceptions import CloudifyCliError
 
 
+def _open_interactive_shell(host_string, command=''):
```

Comment:
I would have changed the command arg to tmux=False
so the api is: _open_interactive_shell(host_string, tmux=False)
and the tmux command defined inside the method...